dockerfiles/postgres/Dockerfile

```diff
@@ -16,5 +16,5 @@ RUN chmod +x /var/lib/pgsql/init-che-user-and-run.sh
 USER postgres
 ADD --chown=postgres postgresql.conf.debug /opt/app-root/src/postgresql-cfg/
 ADD init-debug.sh /opt/app-root/src/postgresql-pre-start/
-RUN chgrp -R 0 /opt/app-root/src/postgresql-cfg/ && chmod -R g+rwX /opt/app-root/src/postgresql-cfg/
+# RUN chgrp -R 0 /opt/app-root/src/postgresql-cfg/ && chmod -R g+rwX /opt/app-root/src/postgresql-cfg/
```

Comment:
is this comment expected?

Reply:
in general wondering if we still need dockerfile for postgress since it was removed in 7.62.0 - https://che.eclipseprojects.io/2023/03/20/@ilya.buziuk-decommissioning-postgresql-database.html

Reply:
This is a temporary solution, it is going to be removed by https://github.com/eclipse-che/che-server/pull/468

Reply:
I'm still not clear with this change as part of podman switch.
Do we really need to update postrgres Dockerfile as part of this PR? 

@SDawley @nickboldt is #468 still actual which was opened in March

Reply:
@ibuziuk @nickboldt @SDawley I closed the https://github.com/eclipse-che/che-server/pull/468 in favor of https://github.com/eclipse-che/che-server/pull/573. This is the same PR but I merged the main branch into it, since the original PR was located in the personal fork.